sdk/keyvault/azure-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/ListOperationsAsync.java

```diff
@@ -35,11 +35,11 @@ public static void main(String[] args) throws InterruptedException {
         keyAsyncClient.createEcKey(new EcKeyCreateOptions("CloudEcKey")
                 .expires(OffsetDateTime.now().plusYears(1)))
                 .subscribe(keyResponse ->
-                        System.out.printf("Key is created with name %s and type %s \n", keyResponse.value().name(), keyResponse.value().keyMaterial().kty()));
+                        System.out.printf("Key is created with name %s and type %s \n", keyResponse.name(), keyResponse.keyMaterial().kty()));
 
         Thread.sleep(2000);
 
-        keyAsyncClient.createRsaKey(new RsaKeyCreateOptions("CloudRsaKey")
+        keyAsyncClient.createRsaKeyWithResponse(new RsaKeyCreateOptions("CloudRsaKey")
```

Comment:
We should probably show a detailed sample for withResponse, processing/printing the headers info.

Reply:
But that might lead to a lot of similar-looking samples 🤷‍♀

Reply:
We can just update the existing sample, and make it print the response headers info along with the returned key details.

Reply:
I think we should have samples which show the T response type methods as that is the more used one. I will update these examples in #4549 to use T response methods.